EarlGrey/Exception/GREYDefaultFailureHandler.m

```diff
@@ -44,18 +44,49 @@ - (void)setInvocationFile:(NSString *)fileName andInvocationLine:(NSUInteger)lin
 
 - (void)handleException:(GREYFrameworkException *)exception details:(NSString *)details {
```

Comment:
Can you please explain in detail why are you making this change? What's this going to achieve? If all you need is some additional fields printed, just include those as changes.